js/index.js

```diff
@@ -266,6 +266,7 @@ demoBtn.addEventListener('click', generateDemo);
 //timer controls
 setTimerBtn.addEventListener('click', () => {    
     let minutes, seconds;        
+    // Comment
```

Comment:
This callback is very similar to `formatTimer`.  If you wanted, you could call the `formatTimer` function in the callback.  For example:
```
setTimerBtn.addEventListener('click', () => {
  formatTimer()
  if(timerDuration.value > 10)
  ...
});

Reply:
Ah, yep. The formatTimer function was a part of some refactoring where I broke it out of an old function that did a bunch of random things. Just missed that bit.